src/DataValues/DecimalValue.php

```diff
@@ -355,10 +355,11 @@ public function getArrayValue() {
 	 *
 	 * @deprecated since 0.8.3. Static DataValue::newFromArray constructors like this are
 	 *  underspecified (not in the DataValue interface), and misleadingly named (should be named
-	 *  newFromArrayValue). Instead, use DataValue builder callbacks in @see DataValueDeserializer.
+	 *  newFromArrayValue). Instead, use the constructor or use DataValue builder
+	 *  callbacks. (e.g. @see DataValueDeserializer)
```

Comment:
@aude, I know that. Is this meant as a response to something @manicki or I said?